### ISSUE TITLE
After install generator appears to not be installed

### ISSUE THREAD

**Issue by mikecfisher** [ironSource/node-generator-bare-react#1]
I'm trying to setup this generator on my system. After running `npm install -g generator-bare-react` everything installs ok. Then after running `yo bare-react` I get an error message stating. 

```
Error bare-react

You don't seem to have a generator with the name bare-react installed.
You can see available generators with npm search yeoman-generator and then install them with npm install [name].
To see the 48 registered generators run yo with the `--help` option.
```

**Comment by mikecfisher** [comment]
FYI I'm using npm v3

**Comment by vweevers** [comment]
Thanks for the report! Fixed in 1176a5a, published 1.1.1.
